readme.rst

```diff
@@ -5,23 +5,44 @@ See the `Sphinx and Read the Docs <https://canonical-documentation-with-sphinx-a
 
 Then go through the following sections to use this starter pack to set up your docs repository.
 
-Download and install
---------------------
+Set up your documentation repository
+------------------------------------
 
-* *Incorporate this starter pack into an existing code repository* - copy all
-  the files from this repository into your project's directory structure,
-  and rename this directory to ``docs`` or similar.
+You can either create a standalone documentation project based on this repository or include the files from this repository in a dedicated documentation folder in your code repository.
 
-* *Start a standalone documentation project* - clone this locally and start
-  working.
+**Note:** We're planning to provide the contents of this repository as an installable package in the future, but currently, you need to copy and update the required files manually.
 
-In documentation directory, run::
+Standalone documentation repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create a standalone documentation repository, clone this starter pack repository, `update the configuration <#configure-the-documentation>`_, and then commit all files to your own documentation repository.
+
+You don't need to move any files, and you don't need to do any special configuration on Read the Docs.
+
+Documentation in the code repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To add your documentation to your code repository, create a dedicated documentation folder in your repository (for example, ``docs``).
+
+Copy the files from this starter pack repository into this ``docs`` folder.
```

Comment:
"Copy all the files and folders from this repository into this `docs` folder, **with the exception of the `.git` folder**."